configs/config/benchmark/linear_image_classification/imagenette/eval_resnet_8gpu_transfer_imagenette_160.yaml

```diff
@@ -0,0 +1,111 @@
+# @package _global_
```

Comment:
is it possible to move this file to `benchmark/linear_image_classification/datasets/imagenette_160.yaml` ? and only keep the relevant changes in the file ? we can compose the main benchmark config and dataset config together :)